nodes/pods/run_once_duration_override/run-once-duration-override-release-notes.adoc

```diff
@@ -14,14 +14,14 @@ These release notes track the development of the {run-once-operator} for {produc
 
 For an overview of the {run-once-operator}, see xref:../../../nodes/pods/run_once_duration_override/index.adoc#run-once-about_run-once-duration-override-about[About the {run-once-operator}].
 
-[id="run-once-duration-override-operator-release-notes-1-2-0"]
-== {run-once-operator} 1.2.0
+[id="run-once-duration-override-operator-release-notes-1-3-0_{context}"]
+== {run-once-operator} 1.3.0
 
-Issued: 16 October 2024
+Issued: 9 July 2025
+//Advisory link provided the day of the release.
+The following advisory is available for the {run-once-operator} 1.3.0: (link:https://access.redhat.com/errata/RHBA-2025:10725[RHBA-2025-10725])
```

Comment:
This isn't rendering correctly in preview. I think we need to remove the comment or add a carriage return.
```suggestion
Issued: 9 July 2025

//Advisory link provided the day of the release.
The following advisory is available for the {run-once-operator} 1.3.0: (link:https://access.redhat.com/errata/RHBA-2025:10725[RHBA-2025-10725])
```